src/it/resources/com/google/checkstyle/test/chapter3filestructure/toolongpackagetotestcoveragegooglesjavastylerule/InputLineLength.java

```diff
@@ -1,77 +0,0 @@
-package com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule; // ok
-import java.io.*;
-final class InputLineLength
-{
-    // Long line ---------------------------------------------------------------------------------------- //warn
-    // Contains a tab ->    <-
-    // Contains trailing whitespace ->
-
-    // Name format tests
-    //
-    /** Invalid format **/
-    public static final int badConstant = 2;
-    /** Valid format **/
-    public static final int MAX_ROWS = 2;
-
-    /** Invalid format **/
-    private static int badStatic = 2;
-    /** Valid format **/
-    private static int sNumCreated = 0;
-
-    /** Invalid format **/
-    private int badMember = 2;
-    /** Valid format **/
-    private int mNumCreated1 = 0;
-    /** Valid format **/
-    protected int mNumCreated2 = 0;
-
-    /** commas are wrong **/
-    private int[] mInts = new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24}; //warn
-
-    /**
-     * Very long url: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java
-     */
-    public void fooMethod() {}
-
-    /**
-     * Long url without wrapping: http://ftp.dlink.ru/pub/D-Link_Solutions/D-Link_Solutions_for_Business.pdf
-     */
-    public void fooMethodLongFtp() {}
-
-    public void fooLongStringUrl() {
-        String url = "https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java"; //ok
-        processUrl("https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java"); //ok
-        processUrl("some line"
-                + "https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java" //ok
-                + "+ long fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo00000000000o line"); //warn
-        processUrl("Some long foooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo line"); //warn
-        String[] soooooooooooooooooooooooooooooooooooolongfooooooooooooooooooooooooooooooooooooooooooo = { //warn
-            "http://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java", //ok
-            "Some long foooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo line", //warn
-        };
-
-        String fakehttps = "Some long foooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo line"; //warn
```

Comment:
why such inputs are removed ?

Reply:
the section is about checking the line-wrapping of package statement, the column limit rule does not applies on it. So I removed all the other code which was not necessary including this, as this code does not show that we're following the rule.

the following code shows that column limit does not applies to the package statement:

https://github.com/checkstyle/checkstyle/blob/8420b5438042bab380146b7ca936e2a36b959a56/src/it/resources/com/google/checkstyle/test/chapter3filestructure/toolongpackagetotestcoveragegooglesjavastylerule/InputPackageStatement.java#L1-L4

and this code shows that package statements are not wrappable:

https://github.com/checkstyle/checkstyle/blob/8420b5438042bab380146b7ca936e2a36b959a56/src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java#L1-L3

Though to show that other code still needs to follow the column limit rule, I have kept some of that example code too:

https://github.com/checkstyle/checkstyle/blob/8420b5438042bab380146b7ca936e2a36b959a56/src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java#L5-L8

All the other code was extra so it was okay to remove it. It does not affect the test coverage of this section. In short we're still testing the section fully and correctly.

Reply:
Ok , good.